examples/openai_embedding_client.py

```diff
@@ -13,11 +15,23 @@
 models = client.models.list()
 model = models.data[0].id
 
-responses = client.embeddings.create(input=[
+input = [
     "Hello my name is",
     "The best thing about vLLM is that it supports many different models"
-],
-                                     model=model)
+]
 
-for data in responses.data:
-    print(data.embedding)  # list of float of len 4096
+responses_float = client.embeddings.create(input=input, model=model, 
+                                           encoding_format="float")
+
+responses_base64 = client.embeddings.create(input=input, model=model, 
+                                            encoding_format="base64")
+
+
+decoded_responses_base64_data = []
+for data in responses_base64.data:
+    decoded_responses_base64_data.append(
+        np.frombuffer(base64.b64decode(data.embedding), dtype="float").tolist()
```

Comment:
Hmm, so OpenAI doesn't perform automatic decoding anymore?

Reply:
> Hmm, so OpenAI doesn't perform automatic decoding anymore?

Yes, from https://github.com/openai/openai-python/blob/main/src/openai/resources/embeddings.py#L98, it seems because we explicitly specify the `encoding = base64 or float`, it does not run the rest of the logics in `parse` func

Reply:
Ah, I see. Can you add this example file to be explicitly tested in CI?

Reply:
Also, the linter is currently failing, please use `bash format.sh` to fix such errors locally.

Reply:
I will do the unit testing for this one

Reply:
Too late - a test has already been added to this PR. Unless you have more tests?

Reply:
No no, seems alright!